src/client.rs

```diff
@@ -3,7 +3,9 @@ use crate::{
     indexes::*,
     key::{Key, KeyBuilder},
     request::*,
-    tasks::{async_sleep, Task},
+    task_info::TaskInfo,
+    tasks::*,
+    utils::async_sleep,
```

Comment:
Why this one is import you don't seem to use it?

Reply:
Since it is linked to the wasm mystery, I just copy pasted code that was already present and will look into this once we start digging more into the wasm